exoctk/contam_visibility/sossContamFig.py

```diff
@@ -6,330 +6,220 @@
 from bokeh.layouts import gridplot
 from bokeh.plotting import figure
 from bokeh.models import Range1d, LinearColorMapper, Label
+from bokeh.models.widgets import Panel, Tabs
 from bokeh.palettes import inferno
 import numpy as np
 
 from . import visibilityPA as vpa
 
+TRACES_PATH = os.path.join(os.environ.get('EXOCTK_DATA'),  'exoctk_contam', 'traces')
+
+disp_nircam = 0.001 # microns

```

Comment:
Definitely need to add pysiaf since these can change in the future. According to the maintainers, it shouldn't be by much to change the visibility but we should still keep it in mind.